raft/raft.go

```diff
@@ -718,14 +717,10 @@ func stepLeader(r *raft, m pb.Message) {
 				return
 			}
 			r.abortLeaderTransfer()
-			r.logger.Infof("%x [term %d] abort transfer leadership to %x", r.id, r.Term, lastLeadTransferee)
+			r.logger.Infof("%x [term %d] abort previous transferring leadership to %x", r.id, r.Term, lastLeadTransferee)
 		}
 		if leadTransferee == r.id {
-			if lastLeadTransferee == None {
-				r.logger.Debugf("%x is already leader. Ignored transfer leadership to %x", r.id, r.id)
-			} else {
-				r.logger.Debugf("%x abort transfer leadership to %x, transfer to current leader %x.", r.id, lastLeadTransferee, r.id)
-			}
+			r.logger.Debugf("%x is already leader. Ignored transferring leadership to self", r.id)
```

Comment:
I think the additional variable is fine. It makes the code more readable. And yes, I want to combine the two cases since I feel it is clear enough with the logging at line720. Logging it again seems to be too verbose and lead to confusion.